### PR TITLE
Portfolio : Mobile first-First section

### PR DESCRIPTION
**Pull request by HossainAraf**
This is a portfolio webpage containing a single page. The following technologies were used:

1.  Gig & Github
2. Linters : stylelint, hint
3. HTML
4. CSS, CSS-grid, Flex-box

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+  .menu:hover li,
+  .toggle-button:hover + .menu li {
+    flex-basis: calc(99% / 3);
+    padding-right: 5%;
+  }
+}
+
+/* Media query for desktop screens */
+@media (min-width: 768px) {
```
Comment (joskalenda) [inline style.css]: - [ ] Great initiative, @HossainAraf, but I would ask you to remove the media query for now as it is the part of next milestone, see? desktop version

File: style.css [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+/* Header with Logo & Navbar */
+.header {
+  display: flex;
```
Comment (joskalenda) [inline style.css]: - [ ] As highlighted in red, let me explain how each part for a better understanding, remember, you need to match the figma template as closely as you can, see the header design, your hamburger is in red and not sharing the same design as the original template, and again, your header as a small background that should not be present in the design, please modify them on the go.
- [ ] About your text, the hero's main title is smaller than the original one.
- [ ] The paragraph color is more black than the original one, I think modifying the color to match the figma template is what is required here.
- [ ] the `let's connect` text should be in capital letters
- [ ] Your social media icons are not well-spaced, please update your padding to make the content breath abit
- [ ] The last one is your background image, please position it as you can see it on the original design on the left

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @HossainAraf , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. Green linters ✅ 
- [ ].. Good HTML code ✅ 
- [ ].. Good readme file✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

- [ OPTIONAL ] You have 30 modified files but only in one commit, this is not a best practice at all, in case you need to check your history for a certain modification or whatever, in your case, it will be more complicated,  take the habit of committing at each time you finish a single part, please


Cheers, and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+<body>
+  <header class="header">
+    <div id="status-bar" class="flex">
+      <div id="time">9:41</div>
+      <div class="flex" id="bars">
+        <img src="img/Cellular-Connection.png" alt="">
+        <img src="img/Wifi.png" alt="">
+        <img src="img/Battery.png" alt="">
+      </div>
+    </div>
```
Comment (zieeco) [inline index.html]: - Hey, great job with fixing the changes as suggested by the previous reviewer, however, you've added a part of the design which you're not meant to implement as this part was a screenshot taken from a mobile device. Please, kindly remove it.

image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @HossainAraf,




Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Nice implementing the required changes :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HossainAraf** [comment]
Ok ,Thanks

Best Regards
Md Arafat Hossain

**Review by ademibowale — APPROVED** [review]
Hi @HossainAraf  , 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HossainAraf** [comment]
Thank you.

Best Regards
Md. Arafat Hossain

**Pull request merged**